crates/bevy_ecs/src/error/handler.rs

```diff
@@ -1,75 +1,150 @@
-use crate::{component::Tick, error::BevyError, resource::Resource};
-use alloc::borrow::Cow;
+use core::fmt::Display;
+use std::sync::OnceLock;
```

Comment:
```suggestion
use bevy_platform_support::sync::OnceLock;
```
That _should_ just work.